client/src/App.js

```diff
@@ -20,9 +21,10 @@ export default class App extends Component {
 
   render() {
     return (
-      <div>
+      <div className="app">
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route path="/" component={MovieList} exact/>
+        <Route path="/movies/:id" component={Movie} exact/>
```

Comment:
this will work since there are no further extension of this route/nested routes like `/movies/:id/director` or something, but the `exact` is superfluous here since there are no other Route paths you build that could overlap with this. the general rule of thumb is to only include the exact attribute if that is the case